pkg/server/server_test.go

```diff
@@ -75,6 +80,17 @@ func fbcJsonCache(catalogDir, cacheDir string) (cache2.Cache, error) {
 	return store, nil
 }
 
+func fbcJsonCacheFromFs(catalogFS fs.FS, cacheDir string) (cache2.Cache, error) {
+	store := cache2.NewJSON(cacheDir)
```

Comment:
nit: I know it's also pre-existing, but prefer a more human-friendly import alias to `cache2`

Reply:
I pondered this one, but whatever logic drove the name is lost in time.  The best correlation I had was that "this was the second server cache created in the test".  
This is easily enough changed to `fbccache`